web/stats/aoc_structures.json

```diff
@@ -1169,10 +1169,10 @@
       "hp": "2750",
       "ra": "-",
       "at": "-",
-      "ar": "10/10",
-      "GA": "+1250 HP with Fortified Wall<br />(sometimes gate parallel to the bottom right edge has 0:30 bt)",
+      "ar": "6/6",
```

Comment:
You are right, that most gates have 6/6, but gates parallel with the bottom right edge have 10/10. Could we add that rather to the note? Or did you extra testing with that "special gate"?